conmon-rs/server/src/attach.rs

```diff
@@ -237,7 +237,11 @@ impl Attach {
                                 e.raw_os_error().context("get OS error")?
                             ),
                         },
-                        _ => {}
+                        _ => {
```

Comment:
We are missing the n == 0 case. Being explicit with that case should be able to replace the '_' case.

Reply:
Ah, I see your note on the other PR